### PR TITLE
Use urllib instead of requests

### PR DESCRIPTION
**Pull request by nre-ableton**
This allows users to run `fetch_jars.py` without a virtualenv, which also means we can eliminate this project's `requirements.*` files. Also, the code is even a bit simpler with `urllib` instead of `requests`, since we are not doing any fancy stuff with the `requests` library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ issues and more.
 ### Running with Python
 
 To use `groovylint` as a standalone Python script, you should first clone the repository
-somewhere on your hard drive. Pipenv is required to set up `groovylint`, but not to run
-it. After running `pipenv sync`, you can run the `fetch_jars.py` script:
+somewhere on your hard drive. Next, run the `fetch_jars.py` script to fetch CodeNarc's JAR
+files:
 
 ```bash
 $ pipenv run fetch_jars --codenarc 1.2.1 --gmetrics 1.0 --slf4j 1.7.25 \
```
Comment (ala-ableton) [inline README.md:16]: `pipenv run` should be removed here.

**Review by ala-ableton — CHANGES_REQUESTED** [review]

File: fetch_jars.py [hunk 2]
```diff
@@ -30,13 +29,7 @@ def download_file(url, output_dir, force=False):
...
 
     logging.debug('Downloading %s to %s', url, output_file_path)
-    response = requests.get(url, stream=True)
-
-    response.raise_for_status()
-
-    with open(output_file_path, mode='wb') as output_file:
-        for chunk in response.iter_content(chunk_size=256):
-            output_file.write(chunk)
+    urllib.request.urlretrieve(url, output_file_path)
```
[the remark this answers is not in the document]
Comment (nre-ableton) [reply fetch_jars.py]: Thanks, I didn't realize that. Fixed in a0446c1.

File: README.md (hunk 1 above)
Comment (nre-ableton) [reply README.md]: Fixed in 548f703. Nicely, this all fits on a single line now.

File: README.md [hunk 3]
```diff
@@ -9,12 +9,11 @@ issues and more.
...
 To use `groovylint` as a standalone Python script, you should first clone the repository
-somewhere on your hard drive. Pipenv is required to set up `groovylint`, but not to run
-it. After running `pipenv sync`, you can run the `fetch_jars.py` script:
+somewhere on your hard drive. Next, run the `fetch_jars.py` script to fetch CodeNarc's JAR
+files:
 
 ```bash
-$ pipenv run fetch_jars --codenarc 1.2.1 --gmetrics 1.0 --slf4j 1.7.25 \
-  --output-dir ./resources
+$ fetch_jars --codenarc 1.2.1 --gmetrics 1.0 --slf4j 1.7.25 --output-dir ./resources
```
Comment (ala-ableton) [inline README.md:16]:
```suggestion
$ ./fetch_jars.py --codenarc 1.2.1 --gmetrics 1.0 --slf4j 1.7.25 --output-dir ./resources
```
or
```suggestion
$ /path/to/groovylint/fetch_jars.py--codenarc 1.2.1 --gmetrics 1.0 --slf4j 1.7.25 --output-dir ./resources
```
to match the following code block.

**Review by ala-ableton — APPROVED** [review]
Once rebased, LGTM 👍🏻

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**